src/asmcnc/comms/logging/log_exporter.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime
+import os
+from asmcnc.production.database import credentials as creds
+import paramiko
+
+WORKING_DIR = 'C:\\SBLogs\\'
+
+export_logs_folder = '/home/pi/exported_logs'
+
+
+def create_log_folder():
+    if not os.path.exists(export_logs_folder) or not os.path.isdir(export_logs_folder):
+        os.mkdir(export_logs_folder)
+
+
+def generate_logs():
+    create_log_folder()
+
+    str_current_time = datetime.now().strftime("%m-%d-%Y-%H-%M-%S")
+
+    log_name = "easycut-exported-logs-" + str_current_time + ".txt"
```

Comment:
Is there a reason not to use the smartbench_logs.txt path (/home/pi/smartbench_logs.txt) that is generated when users download logs with onto USB? Just thinking it would be good to keep all logs in the same place (also for my sanity when I look for them), and it means that any big log files won't get duplicated.

Reply:
time looks useful to have in file as well though

Reply:
no particular reason, but time is helpful. Will add a serial too if possible

Reply:
ok, can you rename it smartbench-exported-logs- (time) - (serial) .txt ? 

then we can make the USB export match the same location later on. Customers don't know that the SW is called easycut, and it can be confusing.